renv.lock

```diff
@@ -1,6 +1,6 @@
 {
   "R": {
-    "Version": "4.2.1",
+    "Version": "4.1.2",
```

Comment:
oh @Jeff-Thompson12, is it okay if this stays consistent with RAA? Just so we don't have to cumbersomely change versions back and forth?

Reply:
That makes sense to me.